src/overlays/actors/ovl_Bg_Hakugin_Post/z_bg_hakugin_post.c

```diff
@@ -21,103 +54,994 @@ const ActorInit Bg_Hakugin_Post_InitVars = {
     (ActorFunc)NULL,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80A9D8B8 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_NONE, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000100, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+typedef struct {
+    f32 unk_00;
+    u8 unk_04;
+} BgHakuginPostUnkStruct3;
+
+static BgHakuginPostUnkStruct3 D_80A9D880[] = {
+    { 1200.0f, false }, { 600.0f, false }, { 900.0f, false }, { 870.0f, false },
+    { 120.0f, false },  { 120.0f, true },  { 200.0f, true },
+};
+
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x00000100, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 276, 0, 20, { 0, 0, 0 } },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80A9D8F8[] = {
+static Color_RGBA8 D_80A9D8E4 = { 250, 250, 250, 255 };
+static Color_RGBA8 D_80A9D8E8 = { 180, 180, 180, 255 };
+static Vec3f D_80A9D8EC = { 0.0f, 0.3f, 0.0f };
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+void func_80A9ACD0(void* arg0) {
```

Comment:
```suggestion
void func_80A9ACD0(BgHakuginPostUnkStruct* arg0) {
```

Reply:
I left it as void since bzero wants void. I'd have to put in an extra cast to avoid warnings.